ledger/participant-state/BUILD.bazel

```diff
@@ -23,6 +23,7 @@ da_scala_library(
         "//daml-lf/transaction:transaction_java_proto",
         "//daml-lf/transaction:value_java_proto",
         "//ledger/ledger-api-domain",
+        "//ledger/ledger-api-health",
```

Comment:
Is this needed in `participant-state`?

Reply:
Yup, because I added the `ReportsHealth` trait to `ReadService` and `WriteService`. I didn't want to make it part of _ledger-api-domain_ because it's not _our_ domain concept.

Reply:
👍